pillar/edx/ansible_vars.sls

```diff
@@ -453,11 +335,6 @@ edx:
         - courseware.student_field_overrides.IndividualStudentOverrideProvider
       GIT_IMPORT_STATIC: false
       LOGGING_ENV: lms-{{ edx.edxapp_log_env_suffix}}
-      OAUTH_OIDC_ISSUER: "{{ EDXAPP_LMS_ISSUER }}"
```

Comment:
This should be kept in the base pillar so that OAUTH can be used for sandbox deployments